chromium/pages/options/ux.js

```diff
@@ -5,6 +5,20 @@
 
 "use strict";
 
+if (~navigator.userAgent.indexOf("Android")) {
```

Comment:
?

Reply:
https://wsvincent.com/javascript-tilde/

Reply:
Looks like we can start using `.includes` as well, since we've specified https://github.com/EFForg/https-everywhere/blob/master/chromium/.eslintrc.json#L8

Reply:
Can you do `!== -1` instead, for ease of reading, or `String.includes` since it's supported already.